File: php/example_container_sealine.php
<?php
require('ContainerTrackingApi.php');

$apiWrapper = new ContainerTrackingApi('YOUR_API_KEY');

try {
    $result = $apiWrapper->container('MEDU6965343', 'MSCU');
    print_r(json_decode($result));
} catch (Exception $e) {
    echo 'Error: ' . $e->getMessage() . PHP_EOL;
}

File: python/example_container_sealine.py
from containertrackingapi import ContainerTrackingApi

api_wrapper = ContainerTrackingApi(apikey='YOUR_API_KEY')

try:
    result = api_wrapper.container(container_number="MEDU6965343", sealine="MSCU")
    print(result)
except Exception as e:
    print(e)
